evaluator.py

```diff
@@ -107,14 +161,26 @@ def eval_log(self, log_path: Path) -> None:
                 run["flow_test"] = dataclass_from_dict(IperfOutput, run["flow_test"])
 
             self._eval_flow_test(run["flow_test"])
-            for plugin in run["plugins"]:
-                # TODO: add evaluation for plugins
-                # logger.debug(f'Reading result from plugin {plugin["name"]}')
-                pass
+            for plugin_output in run["plugins"]:
+                # TODO: add evaluation for measure_cpu and measure_power plugins
+                plugin_output = dataclass_from_dict(PluginOutput, plugin_output)
+                if plugin_output.name == "validate_offload":
```

Comment:
could we put string literal "validate_offload" in constant or enum?

Reply:
Done